PowerShell/JumpCloud Commands Gallery/Mac Commands/Mac - Install Remote Assist.md

```diff
@@ -0,0 +1,71 @@
+#### Name
+
+Mac - Install Remote Assist | v1.0 JCCG
+
+#### commandType
+
+mac
+
+#### Command
+
+```
+set -euo pipefail
+
+declare -r REMOTE_DMG_URL="https://cdn.awsstg.jumpcloud.com/TheJumpCloud/jumpcloud-remote-assist-agent/latest/jumpcloud-assist-app.dmg"
+declare -r LOCAL_DMG_TMP_PATH="/tmp/jumpcloud-assist-app.dmg"
+declare -r APP_NAME="Jumpcloud Assist App"
```

Comment:
nit ... Jumpcloud -> JumpCloud

Reply:
True both! But this is used to construct a filename that needs to be in sync with upstream, so it's kind of locked in.